src/service/ActivityEventService.ts

```diff
@@ -3,12 +3,28 @@ import { ActivityEvent } from "../model/ActivityEvent"
 import { ActivityEventRepository } from "../repository/ActivityEventRepository"
 import { SecurityContext, ActionContext, _verify } from "./Security"
 import jsonata from "jsonata"
-
+import Ajv from 'ajv';
+import api_schema from './api_schema.json'
+  
 export const ActivityEventService = Router()
 ActivityEventService.post("/participant/:participant_id/activity_event", async (req: Request, res: Response) => {
   try {
-    let participant_id = req.params.participant_id
+    let participant_id = req.params.participant_id    
+    var ajv = new Ajv({allErrors: true, unknownFormats: ['int64'], jsonPointers: false, useDefaults:false});
+    ajv.addMetaSchema(require('ajv/lib/refs/json-schema-draft-06.json'));
```

Comment:
We should probably create the `ajv` instance once in `app.ts` and import that instead of creating it every single time an API call is being serviced.